fastevap.f90

```diff
@@ -0,0 +1,206 @@
+!!!!! FAST EVAPORATION ROUTINE !!!
```

Comment:
This sort of works in some regimes, but it's a little flimsy. Maybe we strengthen the warnings?